### ISSUE TITLE
Can’t connect iOS to Bowflex Velocore 16

### ISSUE THREAD

**Issue by dtodd-hub** [cagnulein/qdomyos-zwift#2908]
QZ app shows something is connected and whilst I am not 100pc sure it is the bike, I don’t know what else it might be! On pedalling, nothing happens in the app.

[debug-Mon_Dec_16_09_21_04_2024.log](https://github.com/user-attachments/files/18147671/debug-Mon_Dec_16_09_21_04_2024.log)
![IMG_7032](https://github.com/user-attachments/assets/5bcdf3e2-5e62-4d90-b563-08c36c797c0a)

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Append a debug log**

 Follow this guide https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F
  
  
**Additional context**

**Comment by cagnulein** [comment]
nevermind, i checked the log better and it seems that the iphone is not getting the connection to the bike. are you able to connect to the bike from any other app from the same phone?

**Comment by dtodd-hub** [comment]
Hmmm, maybe it is not possible with this bike model. Slightly confusing messaging on screen (see snips). I wonder if anyone has got it to work with bowflex velocore16?

![IMG_7044](https://github.com/user-attachments/assets/5018b6f4-438c-4296-ac41-01bd813ad009)
![IMG_7043](https://github.com/user-attachments/assets/8a6a036d-4fbb-4afb-baa6-24b3b34834f8)

**Comment by cagnulein** [comment]
Does bowflex have its own app? Did you try it to the same phone?

**Comment by dtodd-hub** [comment]
Yes. It is called JRNY and message on that app is the same. This bike model not available for Bluetooth connection. Workouts on the bike must sync via internet to the app.
![IMG_7047](https://github.com/user-attachments/assets/ed3858f2-f21d-4820-b74d-d6d5901352be)

**Comment by dtodd-hub** [comment]
best I can find on the topic. Not sure I trust myself going down this route!
https://xdaforums.com/t/bowflex-velocore.4252069/

**Comment by cagnulein** [comment]
it seems interetsting! in case you can unlock it, i can create an app that listen to the tcp port 9999 as mentioned in the thread and i can send the metrics directly in qz

Let me know!

**Comment by dtodd-hub** [comment]
Just an update to say that I have now connected to the bike via the Zwift app, but not plucked up courage to try unlocking the bike!
